main.go

```diff
@@ -7,7 +7,7 @@ import (
 
 func main() {
 	plugin := &plugin.PermPlugin{}
-	response := command.GeneratePlugin(command.Read(), plugin, ".pb.perm.go")
+	response := command.GeneratePlugin(command.Read(), plugin, ".pb.atlas.query.perm.go")
```

Comment:
I think this changes are better to be done in scope of plugin remaning (in case this is still planned)